Library/Homebrew/utils/diskusage.rb

```diff
@@ -0,0 +1,57 @@
+require "find"
+
+module Utils
+  class DiskUsage
```

Comment:
> You could make a cache in the method using an instance or class variable.

Change available on `brew-cleanup` branch in my fork of `Homebrew`. You can view it [here](https://github.com/rakkesh/homebrew/commit/81db726f7dd592667e60ca24099afe5f8a8386ab).